lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -526,8 +527,7 @@ ArrayAttr TypeLoweringVisitor::filterAnnotations(
       retval.push_back(newAnno.getDict());
       continue;
     }
-    if (Annotation(opAttr).getClass() ==
-        "firrtl.transforms.DontTouchAnnotation") {
+    if (Annotation(opAttr).getClass() == dontTouchAnnoClass) {
```

Comment:
Not the same as the original, but cleaner:
```suggestion
    if (Annotation(opAttr).isClass(dontTouchAnnoClass)) {
```